vllm/distributed/parallel_state.py

```diff
@@ -115,6 +115,11 @@ def init_distributed_environment(
                 local_rank = rank
         global _LOCAL_RANK
         _LOCAL_RANK = local_rank
+
+        if world_size == 1:
+            # Avoid to warmup if world size is 1, to be compatible with vGPU.
```

Comment:
```suggestion
            # Avoid to warmup if world size is 1, to be compatible with vGPU.
            # see https://github.com/vllm-project/vllm/issues/4587 for details.
```

please leave pointers for future code readers

Reply:
done